scene/gui/rich_text_label.cpp

```diff
@@ -1470,7 +1470,32 @@ float RichTextLabel::_find_click_in_line(ItemFrame *p_frame, int p_line, const V
 				}
 			} break;
 		}
-
+		// Adjust for dropcap.
+		int dc_lines = l.text_buf->get_dropcap_lines();
+		float h_off = l.text_buf->get_dropcap_size().x;
+		if (line <= dc_lines) {
+			// if (dc_lines > 0) { // TODO: include as selectable text:
+			// 	Rect2 dc_rect = Rect2(p_ofs + ((rtl) ? Vector2() : Vector2(l.offset.x, 0)), l.text_buf->get_dropcap_size());
+			// 	if (dc_rect.has_point(p_click)) {
+			// 		ItemDropcap *drop = nullptr;
+			// 		if (l.from->type == ITEM_DROPCAP) {
+			// 			drop = static_cast<ItemDropcap *>(l.from);
+			// 		} else {
+			// 			for (Item *it : l.from->subitems) {
+			// 				if (it->type == ITEM_DROPCAP) {
+			// 					drop = static_cast<ItemDropcap *>(it);
+			// 				}
+			// 			}
+			// 		}
+			// 		print_line("DC TEXT", drop->text);
+			// 	}
+			// }
+			if (rtl) {
+				off.x -= h_off;
+			} else {
+				off.x += h_off;
+			}
```

Comment:
Drop cap offset change is correct.

The selection text code should probably be done as a separate PR. This will need to do hit_test on drop cap text buffer and set frame/item/char offset accordingly, and might need some extra changes (I do not remember if drop cap counts towards char offset at all).